lib/sigs/delegated/eth.go

```diff
@@ -0,0 +1,26 @@
+package delegated
+
+import (
+	"fmt"
+
+	"golang.org/x/crypto/sha3"
+)
+
+// EthAddressFromPubKey returns the Ethereum address corresponding to an
+// uncompressed secp256k1 public key.
+//
+// TODO move somewhere else, this likely doesn't belong here.
+func EthAddressFromPubKey(pubk []byte) ([]byte, error) {
```

Comment:
Don't we want to also assert the length here?

Reply:
This was implemented in recent commits.